lib/elixir/test/elixir/module/types/descr_test.exs

```diff
@@ -674,6 +674,10 @@ defmodule Module.Types.DescrTest do
       assert subtype?(f, fun([none(), integer()], term()))
       assert subtype?(fun([none(), number()], atom()), f)
       assert subtype?(fun([tuple(), number()], atom()), f)
+
+      # (none, float)->atom is not a subtype of (none, integer)->atom because float has an empty intersection with integer
```

Comment:
```suggestion
      # (none, float -> atom) is not a subtype of (none, integer -> atom)
      # because float has an empty intersection with integer.
```